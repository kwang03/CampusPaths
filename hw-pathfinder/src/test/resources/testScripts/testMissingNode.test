#Tests Dijkstra's when nodes are missing

CreateGraph myGraph

AddNode myGraph A

AddNode myGraph B

FindPath myGraph A C

FindPath myGraph C D