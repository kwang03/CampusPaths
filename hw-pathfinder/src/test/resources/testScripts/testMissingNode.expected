#Tests Dijkstra's when nodes are missing

created graph myGraph

added node A to myGraph

added node B to myGraph

unknown: C

unknown: C
unknown: D